### PR TITLE
Fix images in Wiki repo not showing in Gitea Wiki (#4690)

### PR DESCRIPTION
**Pull request by mattzzw**
wikiRaw method in routers calls WikiNameToFilename which unconditionally adds ".md" to the path.
As a result, findEntryForFile will fail finding the corresponding git blob.
I added WikiNameToFilenameRaw to the model used in wikiRaw which omits any additions to the filename.

Signed-off-by: Matthias Wientapper m.wientapper@gmx.de

### CODE REVIEW

**Comment by jonasfranz** [comment]
Please fmt and commit.

**Comment by lunny** [comment]
Tests will be better.

File: models/wiki.go [hunk 1]
```diff
@@ -42,6 +42,12 @@ func WikiNameToFilename(name string) string {
...
 }
 
+// WikiNameToFilenameRaw converts a wiki name to its corresponding raw filename.
+func WikiNameToFilenameRaw(name string) string {
+	name = strings.Replace(name, " ", "-", -1)
+	return url.QueryEscape(name)
+}
+
 // WikiFilenameToName converts a wiki filename to its corresponding page name.
 func WikiFilenameToName(filename string) (string, error) {
```
Comment (lafriks) [inline models/wiki.go]: `WikiNameToFilename` should reuse `WikiNameToFilenameRaw`

Comment (mattzzw) [reply models/wiki.go]: Thanks for the review.
Now that I look at the code of models/wiki.go again I can't see the difference between `WikiNameToFilenameRaw`and `WikiNameToSubURL`. I think I totally duplicated a method that was already in place... Doh.
I will fix that.

**Review by lafriks — APPROVED** [review]

**Comment by mattzzw** [comment]
Gentle bump. 😃 
Can I do anything to make this simple PR acceptable?

**Comment by lunny** [comment]
@mattzzw as I mentioned before could you add some tests that will let review easier.

**Comment by zeripath** [comment]
I think this has been fixed by #5814

**Pull request closed without merging**